news/2023/2023-05-10-skinning-contest-icons-of-an-era.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: post
+title: "Skinning Contest: Icons of an Era"
+date: 2023-05-10 12:00:00 +0000
+---
+
+Celebrate 15 years of osu! by expressing your creativity through osu! skinning!
+
+![](https://assets.ppy.sh/contests/178/header.jpg)
+
+Over the course of osu!'s history there have been many memories made, whether that'd be related to maps that would become legendary within the community, plays that to this day remain unbeaten and are considered the best of all time, or the various community-made creations that have been popularised throughout the years.
+
+We wanted this contest's theme to not only centre around some of those memories, but also provide plenty of material that could inspire the skinning community to take part and provide us with some amazing skins!
+
+## Theme: Icons of an Era
+
+This year's theme, **Icons of an Era**, aims to spark the community's imagination by creating art around various iconic osu! beatmaps that were crafted and risen to glory throughout osu!'s lifetime, as well as the many either retired, official or community-made osu! mascots that have been introduced over the course of the past 15 years.
```

Comment:
Probably shouldn't write "this years theme" as we plan to have a second contest this year